packages/ember-intl/package.json

```diff
@@ -35,21 +35,21 @@
     "test": "ember test --test-port=0"
   },
   "dependencies": {
-    "@babel/core": "^7.25.2",
-    "@formatjs/icu-messageformat-parser": "^2.7.8",
-    "@formatjs/intl": "^2.10.4",
+    "@babel/core": "^7.25.8",
+    "@formatjs/icu-messageformat-parser": "^2.7.10",
+    "@formatjs/intl": "^2.10.8",
     "broccoli-caching-writer": "^3.0.3",
     "broccoli-funnel": "^3.0.8",
     "broccoli-merge-trees": "^4.2.0",
     "broccoli-source": "^3.0.1",
     "calculate-cache-key-for-tree": "^2.0.0",
     "cldr-core": "^45.0.0",
-    "ember-auto-import": "^2.7.4",
+    "ember-auto-import": "^2.8.1",
     "ember-cli-babel": "^8.2.0",
     "ember-cli-typescript": "^5.3.0",
     "eventemitter3": "^5.0.1",
     "extend": "^3.0.2",
-    "intl-messageformat": "^10.5.14",
+    "intl-messageformat": "^10.7.0",
```

Comment:
Only the ordering in import statements was changed.

- https://github.com/formatjs/formatjs/blob/intl-messageformat%4010.7.0/packages/intl-messageformat/CHANGELOG.md#1070-2024-10-12
- https://github.com/formatjs/formatjs/commit/12f94f778c6453a75cc7a3c7ac0fc587b6f4a157